disk/datadog_checks/disk/disk.py

```diff
@@ -16,6 +16,9 @@
 from datadog_checks.base.utils.subprocess_output import SubprocessOutputEmptyError, get_subprocess_output
 from datadog_checks.base.utils.timeout import TimeoutException, timeout
 
+if platform.system() == 'Windows':
+    import win32wnet
+
 if platform.system() == 'Windows':
```

Comment:
We check twice for the platform, why not do it in a single if?

```suggestion
if platform.system() == 'Windows':
    import win32wnet

```

Reply:
could do that.  I was going for making the change as localized as possible so as to not break anything else.